extensions/amp-analytics/0.1/session-manager.js

```diff
@@ -113,9 +113,7 @@ export class SessionManager {
       !isSessionExpired(this.sessions_[type])
     ) {
       this.sessions_[type] = this.updateSession_(this.sessions_[type]);
-      if (opt_processing) {
-        opt_processing(this.sessions_[type]);
-      }
+      opt_processing?.apply(this.sessions_[type]);
```

Comment:
Should this be `opt_processing?.(this.sessions_[type])`? The first arg to `apply` is the `this` param

Reply:
~facepalm~ 

Ty for the catch.